pkg/alertmanager/amcfg.go

```diff
@@ -1382,18 +1383,24 @@ func (r *receiver) sanitize(amVersion semver.Version, logger log.Logger) error {
 }
 
 func (ogc *opsgenieConfig) sanitize(amVersion semver.Version, logger log.Logger) error {
-	actionsAndEntityAllowed := amVersion.GTE(semver.MustParse("0.24.0"))
-	if ogc.Actions != "" && !actionsAndEntityAllowed {
+	actionsAndEntityAndUpdateAlertsAllowed := amVersion.GTE(semver.MustParse("0.24.0"))
```

Comment:
(nit) shorter variable name
```suggestion
	atLeastV0_24 := amVersion.GTE(semver.MustParse("0.24.0"))
```